src/Core/Components/DataGrid/FluentDataGridCell.razor.cs

```diff
@@ -37,6 +43,18 @@ public partial class FluentDataGridCell<TGridItem> : FluentComponentBase
     [CascadingParameter(Name = "OwningRow")]
     public FluentDataGridRow<TGridItem> Owner { get; set; } = default!;
 
+    /// <summary>
+    /// Gets or sets the owning <see cref="FluentDataGrid{TItem}"/> component
+    /// </summary>
+    [CascadingParameter]
+    private InternalGridContext<TGridItem> GridContext { get; set; } = default!;
+
+    protected string? StyleValue => new StyleBuilder(Style)
+       .AddStyle("height", $"{GridContext.Grid.ItemSize}px", () => GridContext.Grid.Virtualize && Owner.RowType == DataGridRowType.Default)
```

Comment:
I don't know why the `ItemSize` is `float` but using this condition, you have to specify the `en-US` culture.
Or maybe better: `$"{GridContext.Grid.ItemSize:0}px"` to take only the integer part.